src/TableTransforms.jl

```diff
@@ -37,6 +37,7 @@ export
   DropMissing,
   Rename,
   Coalesce,
+  Replace,
```

Comment:
Move to before Coalesce

Reply:
OK

Reply:
Made in [9c5c293](https://github.com/JuliaML/TableTransforms.jl/pull/42/commits/9c5c293c10346a2cb1eddf7f2bf11923779906de)